opentelemetry_collector/receiver/vmagereceiver/vm_age_collector.go

```diff
@@ -89,7 +99,17 @@ func (collector *VMAgeCollector) StartCollection() {
 }
 
 func (collector *VMAgeCollector) setupCollection() {
-	collector.parseBuildDate()
+	var err error
+
+	collector.parsedBuildDate, err = collector.parseDate(collector.buildDate)
+	collector.buildDateError = (err != nil)
+
+	collector.parsedVMStartTime, err = collector.parseDate(collector.vmStartTime)
```

Comment:
Since there isn't an error metric for these values, it would be good to log any parsing errors.